examples/ltc_backend/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+###########################################################################
```

Comment:
this is awesome. is there a way we can get this example running under CI? It has a pretty complex build surface area, and seems prone to breakage. I would personally put it somewhere inside the main tree (not examples) and write some small python unit tests that invoke it.

Reply:
> I would personally put it somewhere inside the main tree (not examples)
Do you want the example backend to be moved outside the examples directory? This stuff should already be built automatically with the rest of Torch-MLIR, so I would assume that it should just build with CI (once CI isn't broken at least...)

We previously added e2e tests, which relies on this example backend, so in theory this should be covered by CI tests